Mathlib/Algebra/BigOperators/Sym.lean

```diff
@@ -21,3 +27,17 @@ theorem Finset.sum_sym2_filter_not_isDiag {ι α} [LinearOrder ι] [AddCommMonoi
     simp [and_assoc]
   · rintro ⟨⟨i₁, j₁⟩, hij₁⟩
     simp
+
+theorem sum_range_count_of_mem_sym {α} [DecidableEq α] {m : ℕ} {k : Sym α m} {s : Finset α}
+    (hk : k ∈ s.sym m) : (∑ i ∈ s, count i k) = m := by
+  simp_rw [← k.prop, ← toFinset_sum_count_eq, eq_comm]
+  apply sum_subset_zero_on_sdiff _ _ (fun _ _ ↦ rfl)
+  · intro i hi
+    simp only [Sym.val_eq_coe, mem_toFinset, Sym.mem_coe] at hi
+    exact mem_sym_iff.mp hk i hi
+  · intro _ hx
+    simp only [Sym.val_eq_coe, mem_sdiff, Finset.mem_range, mem_toFinset, Sym.mem_coe] at hx
+    simp only [count_eq_zero, Sym.mem_coe]
+    exact hx.2
```

Comment:
I prefer to see exactly which `_` are being left as goals (as it was, I had to figure it out for myself). Also, let's invoke some automation here!
```suggestion
  refine sum_subset_zero_on_sdiff ?_ ?_ (fun _ _ ↦ rfl)
  all_goals aesop
```

Reply:
I had to change the `refine` slightly; otherwise `aesop` did not close the first goal:
```
refine sum_subset_zero_on_sdiff (fun _ _ ↦ ?_) ?_ (fun _ _ ↦ rfl)
```